src/main/java/seedu/address/logic/commands/Command.java

```diff
@@ -13,8 +13,9 @@
      * Executes the command and returns the result message.
      *
      * @param model {@code Model} which the command should operate on.
-     * @return feedback message of the operation result for display
-     * @throws CommandException If an error occurs during command execution.
+     * @return      results from executing the command.
+     *
+     * @throws      CommandException If an error occurs during command execution.
```

Comment:
I think you missed out the newline in this one

Reply:
Great catch, I'll push the change now